tasks/main.yml

```diff
@@ -13,8 +13,9 @@
 
 - name: install (specified) language packs
   apt:
-    name: "{{ locales_language_packs_present }}"
```

Comment:
Isn't this already the new syntax?

No `loop`s for `apt`

Reply:
I tested it against 2.7.0 and 2.8.0 versions. Both versions does not raise warning with this form.